src/celldega/pre/trx_tile.py

```diff
@@ -4,7 +4,7 @@
 from tqdm import tqdm
 import concurrent.futures
 import pandas as pd
-
+from scipy.sparse import csr_matrix
```

Comment:
Are these changes to trx_tile.py necessary?

Reply:
I see, we are using this function get get the tile boundaries. Lets use this opportunity to switch to using the image to get the boundaries. We can leave this tile boundary export here for now and fix it in the general sense later. We can also get the image size from the dzi files (https://github.com/broadinstitute/celldega_Xenium_Prime_Human_Skin_FFPE_outs/blob/main/Xenium_Prime_Human_Skin_FFPE_outs/pyramid_images/bound.dzi)